libclc/cmake/modules/AddLibclc.cmake

```diff
@@ -42,8 +42,8 @@ endfunction(add_libclc_alias alias target)
 macro(add_libclc_builtin_set arch_suffix)
   cmake_parse_arguments(ARG
     ""
-    "TRIPLE;TARGET_ENV;LIB_DEP;GENERATE_TARGET;PARENT_TARGET"
-    "FILES;ALIASES;COMPILE_OPT"
+    "TRIPLE;TARGET_ENV;LIB_DEP;PARENT_TARGET"
+    "FILES;ALIASES;GENERATE_TARGET;COMPILE_OPT"
```

Comment:
Interesting... is there any difference between old and new code?

Reply:
yes, the first line model single value variable, the second list of variable. Welcome to the wonder of cmake :)

`GENERATE_TARGET` is now a list of targets, in sync with this change https://github.com/intel/llvm/pull/1712/files#diff-b25aa3b585676dbf10fc08d937f595e0R240

Reply:
Ah... My bad. I missed that we have two arguments here. I thought it's a single list...